parquet/README.md

```diff
@@ -0,0 +1,33 @@
+# Parquet Sample data
```

Comment:
Note that the `.DS_Store` file should not be here.

Reply:
deleted in commit 
5f02486